pytorch_lightning/strategies/sharded_spawn.py

```diff
@@ -39,11 +39,10 @@ class DDPSpawnShardedStrategy(DDPSpawnStrategy):
 
     def configure_ddp(self) -> None:
         trainer = self.lightning_module.trainer
-        self.model, optimizers = self._setup_model_and_optimizers(
+        self.model, self.optimizers = self._setup_model_and_optimizers(
```

Comment:
why is this relevant to this PR?

Reply:
this should only be done after the lightning optimizer logic has been cleaned up.